src/Compilers/CSharp/Portable/Binder/Binder_Expressions.cs

```diff
@@ -380,6 +380,11 @@ internal BoundExpression BindToNaturalType(BoundExpression expression, BindingDi
                         result = BindUnconvertedInterpolatedStringToString(unconvertedInterpolatedString, diagnostics);
                     }
                     break;
+                case BoundBinaryOperator unconvertedBinaryOperator:
+                    {
+                        result = RebindSimpleBinaryOperatorAsConverted(unconvertedBinaryOperator, diagnostics);
```

Comment:
>RebindSimpleBinaryOperatorAsConverted [](http://example.com/codeflow?start=33&length=37)

nit: I don't get the "Simple" part. This is specifically a case of a non-simple binary operator (it has unconverted interpolations). 
Maybe `BindUnconvertedInterpolatedStringAddition`? (to follow naming convention of `BindUnconvertedInterpolatedStringToString` above and `IsUnconvertedInterpolatedStringAddition`)

Reply:
`Simple` is an existing term. See `BindSimpleBinaryOperator` in `Binder_Operators.cs`.